app-ios/DroidKaigi2022/DroidKaigi2022.xcodeproj/project.pbxproj

```diff
@@ -166,6 +168,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		FACB42BD28D7208F0034CED3 /* Copy MokoResources */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Copy MokoResources";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "./../scripts/copy_mokoresources.sh\n";
+		};
+/* End PBXShellScriptBuildPhase section */
+
```

Comment:
I'm fine with this for now, but I'll fix it later 🙏🏼

Reply:
Sure! Thanks!